components/pages/home/splitTitle.tsx

```diff
@@ -0,0 +1,458 @@
+import React, { useState, FunctionComponent, useEffect } from "react";
+import styles from "@styles/components/pages/home/splittitle.module.css";
+import { motion } from "motion/react"
+
+
+// This is the order of words appearing
+const words = ["earn", "profit", "maximize", "gain", "grow"]
+
+type ClipPath = { 
+  path: string;       // svg path `d` attribute value. Is always valid Polygon
+  center: number[];   // center of the Polygon to support correct rotation
+}
+
+type SvgSettings = {
+  geometry: React.JSX.Element;  // Geometry to split into pieces
+  size: [number, number];       // (width and height) of viewport and svg tag 
+                                // (TODO(baitcode): should be split into 2 properties)
+  splits: Array<ClipPath>       // Polygon ClipPaths describing shattered shapes
+}
+
+const svgAttributes: Record<string, SvgSettings> = {
```

Comment:
To add in another file as an SVG to `/icons`

Reply:
Pardon. I do not understand what are you trying to say.